### DISCUSSION TITLE
Diskussion

### DISCUSSION THREAD

**Discussion by elstrippi** [elstrippi/WissenschaftlichesArbeiten2023#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaYEEET** [comment]
Das durschnitliche Alter liegt bei Etwa 24,5 Jahren und es Liegt eine sd von etwa 2 vor würde ich anhand der daten Vermuten

**Reply by Lukas107** [reply]
Da wenige Daten vorhanden sind würde ich mal von dem "geraderen" Mittelwert von 25 Jahren ausgehen

**Reply by elstrippi** [reply]
Das ist richtig! Das der Mittelwert der Verteilung ist 25 und die Standardabweichung ist 2

**Reply by DialloKo** [reply]
ich habe unterschiedliche Ergebnisse gehabt wenn ich die ausführe,
zum Beispiel beim Schi-Quadrat  für "categorical_relationship" zwischen  Statistik und MatheLK ist  0.01991653 
das bedeutet hier, dass einen signifikanten Unterschied liegt.

**Reply by MaYEEET** [reply]
komisch ich hab immer das selbe egal wie oft ich dsa ganze ausführe 
![grafik](https://user-images.githubusercontent.com/119424353/221999771-9a518e21-cdf8-4787-a115-e2ace85804bf.png)

**Comment by MaYEEET** [comment]
Bei den Studien Fächern sind Data Science und Statistik ählich oft vertreten mit einem kleinen Vorteil für Data Scince . Informatik ist etwas wenige rvertreten und ganz Abgeschlagen am Ende ist Mathe

**Reply by Lukas107** [reply]
Genauer waren 38% Data Scientists , 22% Informatiker, 5% Mathematiker und 35% Statistiker.
Wenn man wieder von runden Werten ausgeht würde man auf 40% Data Scientists , 20% Informatiker, 5% Mathematiker und 35% Statistiker kommen

**Comment by MaYEEET** [comment]
Daran das Der aufruf von der Berechnung des Kontingenzkoeffizient zu dem zusammen hang von Mathe und dem Mathe Lk einen Fehler ausruft lässt sich erkennen in der Zweiten Variable  nur das Selbe  Element vertreten ist . Ein Aufruf des Ganzen bestätigt das. deswegen vermute ich dsa Mathe Lk und Mathe Studium fest zusammen hängen auch wenn man ds nicht mit einer hohenm sicher heit sagen kann

**Reply by MaYEEET** [reply]
die andern werte bei der Berechnug desKontingenzkoeffizient lassen davon Ausgeh  das etwa 25% Prozent der Data Science- und Statistik- Studenten jeweil keinen Mathe Lk haben. Bei den Informatikern zeugen die werte davon das etwa 18,5% keinen Mathe Lk haben

**Reply by elstrippi** [reply]
Gut erkannt. Wir haben einfach gesagt, dass alle Mathe-Studenten auh im MatheLK waren.

**Comment by Lukas107** [comment]
"Interesse an Programmieren" und "Interesse an Mathe" haben vor allem Werte zwischen 1 und 7 und einige größere Werte die bis 100 hochgehen. Ich vermute mal die Werte sollten eigentlich nur zwischen 1 und 7 liegen und die höheren Werte Fehler sind

**Reply by elstrippi** [reply]
Rihtig. wurde schon korrigiert

**Comment by MaYEEET** [comment]
63 % der STudenten hatten Mathe Lk und 37% nicht

**Reply by MaYEEET** [reply]
Da die Daten simmuliert sind kann man überlegen ob die simmulierten werte nicht eher 60% und 40% sind

**Comment by DialloKo** [comment]
ich habe die Code Kopiert und ausgeführt statt source() funktuion zu benutzen
